### PR TITLE
gconftool2_info: new module

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY
New module to retrieve information from `gconftool-2`.

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME
plugins/modules/system/gconftool2_info.py

### CODE REVIEW

**Comment by russoz** [comment]
WIP for now as it is lacking tests.

File: plugins/modules/system/gconftool2_info.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: xfconf_info
+author:
+    - "Alexei Znamensky (@russoz)"
+short_description: Retrieve XFCE4 configurations
+version_added: 3.5.0
```
Comment (felixfontein) [inline plugins/modules/system/gconftool2_info.py:14]:
```suggestion
version_added: 5.1.0
```

File: plugins/modules/system/gconftool2_info.py [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: xfconf_info
+author:
+    - "Alexei Znamensky (@russoz)"
+short_description: Retrieve XFCE4 configurations
```
Comment (felixfontein) [inline plugins/modules/system/gconftool2_info.py:13]: I think some docs fields need to be updated :)

Comment (russoz) [reply plugins/modules/system/gconftool2_info.py]: Yup, that was a shameless copy-paste. Fixed now.

**Comment by felixfontein** [comment]
@russoz thanks for contributing this!

**Pull request merged**